app/assets/stylesheets/_resources.scss

```diff
@@ -1,4 +1,4 @@
-.resources_index {
+.resources {
```

Comment:
Yes, it fixes the design and has no side effects that I could see.

Middleman gives one of the pages a class of `resources resources_index`, so I gave the appropriate page in Rails a class of `resources`. I could have used `resources_index` too, I suppose.